docs/perf-guide.md

```diff
@@ -214,7 +214,7 @@ DEBUG_DEPTH=5 DEBUG_LEVEL=debug DEBUG=broccoli-merge-trees: ember build
 `[... ConcatWithMaps#782 -> BroccoliMergeTrees#783]` means that the log entry
 occurred in broccoli merge-trees node with id 783, whose parent was a concat
 with maps node with id 782.  These ids are shown in the visualization graph.
-See [Visualization](#Visualization) for details.
+See [Visualization](./perf-guide.md#Visualization) for details.
```

Comment:
this change seems unnecessary 🤔

Reply:
I did this because the link wasn't working, I'm not sure why but you can try to click it here: https://github.com/ember-cli/ember-cli/blob/master/PERF_GUIDE.md

Reply:
Yes, it was, I've made the update.